### PR TITLE
Fixes two tests that fail because of automatic Salesforce translations when creating the scratch org

### PR DESCRIPTION
**Pull request by renatoliveira**
When creating a scratch org Salesforce seems to determine the language/locale automatically. When I created the scratch org I got my org in Brazilian Portuguese instead of US English.

This commit fixes the field label issues by getting them dynamically with .getLabel() and error message issue as well. Both when creating a scratch org in a language other than US English.

Since the `NoDataFoundException` exception is a built-in one, I don't think it is necessary to assert the content of the message. It seems enough to assert that the exception was thrown, and the test already specifies its type instead of handling the generic `Exception` type.

### CODE REVIEW

File: force-app/main/default/classes/MA_ReportServiceTest.cls [hunk 1]
```diff
@@ -63,8 +63,8 @@ private class MA_ReportServiceTest {
         setup();
 
         Map<String, String> expectedColumns = new Map<String, String>{
-            'ACCOUNT_ID' => 'Account ID',
-            'PARENT_ID'  => 'Parent Account ID'
+            'ACCOUNT_ID' => Schema.Account.Id.getDescribe().getLabel(),
```
Comment (douglascayers) [inline force-app/main/default/classes/MA_ReportServiceTest.cls]: Good catch!

File: force-app/main/default/classes/MA_ReportSourceBatchableTest.cls [hunk 2]
```diff
@@ -246,10 +248,11 @@ private class MA_ReportSourceBatchableTest {
             // because the batchable throws the error in the start method
             // then we're getting a synchronous exception caught rather than
             // the job's finish method logging the error asynchronously
-            System.assert( ex.getMessage().containsIgnoreCase( 'The data you’re trying to access is unavailable' ), 'failed for wrong reason' );
-
+            noDataFoundExceptionThrown = true;
         }
 
+        System.assert(noDataFoundExceptionThrown);
```
Comment (douglascayers) [inline force-app/main/default/classes/MA_ReportSourceBatchableTest.cls]: Thanks for the language fixes in this test class, and for [contributing](https://github.com/douglascayers-org/sfdx-mass-action-scheduler/blob/master/CONTRIBUTING.md) to Mass Action Scheduler. 

For the new assert at line 254, for consistency with existing code style, please add a single space after ( and before ) in the method call. 

Additionally, I think there’s still value in the assert having the second argument, 'failed for wrong reason', to provide context around the failure.

File: force-app/main/default/classes/MA_ReportSourceBatchableTest.cls [hunk 3]
```diff
@@ -246,10 +248,11 @@ private class MA_ReportSourceBatchableTest {
             // because the batchable throws the error in the start method
             // then we're getting a synchronous exception caught rather than
             // the job's finish method logging the error asynchronously
-            System.assert( ex.getMessage().containsIgnoreCase( 'The data you’re trying to access is unavailable' ), 'failed for wrong reason' );
-
+            noDataFoundExceptionThrown = true;
```
Comment (douglascayers) [inline force-app/main/default/classes/MA_ReportSourceBatchableTest.cls]: Agree, I like the simplification change here of just relying on the caught exception type and not inspecting the message.

File: force-app/main/default/classes/MA_ReportSourceBatchableTest.cls (hunk 2 above)
Comment (renatoliveira) [reply force-app/main/default/classes/MA_ReportSourceBatchableTest.cls]: I forgot the code style, sorry. I added the message and also another catch for generic exceptions (they shouldn't be catched, so I added an assertion that will fail inside of it). This way we can catch both scenarios where no exception is thrown and the wrong exception is thrown.

Comment (douglascayers) [reply force-app/main/default/classes/MA_ReportSourceBatchableTest.cls]: Thanks, I'm testing the PR and if all is well will merge it in :)

**Pull request merged**